scaffoldmaker/utils/tubemesh2.py

```diff
@@ -0,0 +1,356 @@
+'''
+Utility function for generating tubular mesh from a central line
+using a unit profile.
+Created on 18 Feb, 2019
+
+@author: Mabelle Lin
```

Comment:
We're now removing all @author throughout the code (including my own); the record of who did what is in git.